packages/react-devtools-shared/src/devtools/views/SuspenseTab/SuspenseRects.js

```diff
@@ -109,9 +108,9 @@ function SuspenseRects({
 
 function getDocumentBoundingRect(
   store: Store,
-  shells: $ReadOnlyArray<SuspenseNode['id']>,
```

Comment:
Naming was confusing. The Shell is a slice from Root to the first boundaries. The Root is a single element.